packages/fxa-settings/src/components/Settings/ProgressBar/index.test.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ProgressBar from '.';
+
+describe('Security', () => {
```

Comment:
nit: should this be "ProgressBar"?
"Security" seems a bit... misleading.

Reply:
oh god it is that's a typo sorry